### DISCUSSION TITLE
v2ray保持与一个节点保持连接并且将其他节点视为候选节点？

### DISCUSSION THREAD

**Discussion by weiqi-chen** [v2fly/v2ray-core#1184] [Q&A]
假设说某机场提供了5个节点。这5个hostname:port 各不一样。

所以理论上可以把每一个节点放在一个vnext数组中：outbounds.settings.vnext = { {hostname1, port1}, {hostname2, port2}, ... }

此时v2ray每处理一个新连接，都会把他们均匀地分配给每一个vnext下的服务器。

但是该机场有一个同一用户最大连接数限制，v2ray很快就会与不同的服务器建立过多的连接数，从而使得后续的连接都建立失败了。

请问有没有一个配置办法，让v2ray一旦与某一个节点建立vmess连接后，就不要再和其他候选服务器建立连接。直到当前的服务器连接断开并且无法再新建连接时，自动切换下一个候选服务器呢？

**Comment by BozhanL** [comment]
不知道ObservatoryObject+"type": "leastPing"行不行

**Reply by weiqi-chen** [reply]
"https://www.v2fly.org/config/observatory.html#observatoryobject" 能够“让v2ray一旦与某一个节点建立vmess连接后，就不要再和其他候选服务器建立连接。直到当前的服务器连接断开并且无法再新建连接时，自动切换下一个候选服务器呢？”
